src/core/mac/mac_types.hpp

```diff
@@ -539,7 +541,6 @@ class KeyMaterial : public otMacKeyMaterial, public Unequatable<KeyMaterial>
      */
     bool operator==(const KeyMaterial &aOther) const;
 
-    KeyMaterial(const KeyMaterial &) = delete;
 
```

Comment:
Remove this empty line to resolve [failing pretty check](https://github.com/openthread/openthread/runs/5591707260?check_suite_focus=true):

```
--- src/core/mac/mac_types.hpp	2022-03-17 20:26:59.881163483 +0000
+++ -	2022-03-17 20:27:40.427734038 +0000
@@ -541,7 +541,6 @@
      */
     bool operator==(const KeyMaterial &aOther) const;
 
-
 private:
 #if OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE
     static constexpr KeyRef kInvalidKeyRef = Crypto::Storage::kInvalidKeyRef;
 *** ERROR:  src/core/mac/mac_types.hpp is not pretty.
```